backend/Application/Aggregates/Contract/Jobs/ContractDatabaseImportJob.cs

```diff
@@ -0,0 +1,216 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Application.Aggregates.Contract.Configurations;
+using Application.Aggregates.Contract.Exceptions;
+using Application.Aggregates.Contract.Types;
+using Application.Api.GraphQL.Accounts;
+using Application.Api.GraphQL.Transactions;
+using Microsoft.Extensions.Options;
+
+namespace Application.Aggregates.Contract.Jobs;
+
+internal class ContractDatabaseImportJob : IContractJob
+{
+    private const string JobName = "ContractDatabaseImportJob";
```

Comment:
That could change if a developer by accidant or due to refactor change code. 

By having it as a constant it requires more from the developer to have it change. Since changing this will have job we should do as much as possible to avoid this.

I have added some more comments to code making it clear changing this could trigger rerun of jobs.